docs/Telemetry.md

```diff
@@ -0,0 +1,19 @@
+# Telemetry
+
+## Metrics
+
+If [enabled][metrics feature], [Statsd][statsd] compatible metrics are available to use:
+
+| Name                                           | Type    |
+| ---------------------------------------------- | ------- |
+| zonemaster.rpcapi.requests.\<METHOD>.\<STATUS> | Counter |
+| zonemaster.testagent.tests_started             | Counter |
+| zonemaster.testagent.tests_completed           | Counter |
+| zonemaster.testagent.tests_died                | Counter |
+| zonemaster.testagent.tests_duration_seconds    | Timing  |
+| zonemaster.testagent.running_processes         | Gauge   |
+| zonemaster.testagent.maximum_processes         | Gauge   |
+
```

Comment:
1. I also get
```
zonemaster_testagent_tests_duration_seconds_bucket{le="1"} 0
zonemaster_testagent_tests_duration_seconds_bucket{le="2.5"} 0
zonemaster_testagent_tests_duration_seconds_bucket{le="5"} 0
zonemaster_testagent_tests_duration_seconds_bucket{le="10"} 0
zonemaster_testagent_tests_duration_seconds_bucket{le="15"} 0
zonemaster_testagent_tests_duration_seconds_bucket{le="30"} 4
zonemaster_testagent_tests_duration_seconds_bucket{le="45"} 6
zonemaster_testagent_tests_duration_seconds_bucket{le="60"} 7
zonemaster_testagent_tests_duration_seconds_bucket{le="75"} 8
zonemaster_testagent_tests_duration_seconds_bucket{le="90"} 8
zonemaster_testagent_tests_duration_seconds_bucket{le="105"} 8
zonemaster_testagent_tests_duration_seconds_bucket{le="120"} 8
zonemaster_testagent_tests_duration_seconds_bucket{le="150"} 9
zonemaster_testagent_tests_duration_seconds_bucket{le="180"} 10
zonemaster_testagent_tests_duration_seconds_bucket{le="+Inf"} 10
zonemaster_testagent_tests_duration_seconds_sum 578.0600000000001
zonemaster_testagent_tests_duration_seconds_count 10
```
Shouldn't those be documented above?

2. Could we get some more documentation of what the different metrics mean?

Reply:
Those are metrics generated for the prometheus integration (converting statsd timing metrics to prometheus histogram metrics), they should not be documented as not part of the metrics emitted by Zonemaster. I have updated the PR description for a simpler testing procedure that only output statsd metrics.

I have added some extra documentation, for testing / integration and metrics description.